spec/spec.md

```diff
@@ -151,25 +153,54 @@ Alternatively, one can interpret the encoding rules as a series of transformatio
+to the initial publisher. Consequently, DHT records, including DID DHT Documents, are _independently verifiable_. This independence
+implies that trust in a specific [[ref:Mainline]] or [[ref:Gateway]] server for providing unaltered messages is unnecessary. Instead,
+all clients can, and should, verify messages themselves. This approach significantly mitigates risks associated with other DID methods,
+where a compromised server or [DID resolver](https://www.w3.org/TR/did-core/#choosing-did-resolvers) might tamper with a [[ref:DID Document]]
+which would be undecetable by a client.
+
+Currently, [[ref:Mainline]] exclusively supports the [[ref:Ed25519]] key type. In turn, [[ref:Ed25519]] is required by DID DHT and is 
+used to uniquely identify DID DHT Documents. DID DHT identifiers are formed by concatenating the `did:dht:` prefix with a [[ref:z-base-32]]
+encoded Identity Key, which acts as its [[ref:suffix]]. Identity Keys always have the identifier `0` as both its Verification Method `id` and
+JWK `kid` [[spec:RFC7517]]. While the system requires at least one [[ref:Ed25519]], a DID DHT Document can include any number of additional keys.
+However, these additional key's types ****MUST**** be registered in the [Key Type Index](registry/index.html##key-type-index).
+
+As a unique consequence of the requirement of the Identity Key, DID DHT Documents are able to be partially-resolved without contacting
+[[ref:Maineline]] or [[ref:Gateway]] servers, though it is ****RECOMMENDED**** that deterministic resolution is only used as a fallback mechanism.
+Similarly, the requirement of an Identity Key enables [interoperability with other DID methods](#interoperability-with-other-did-methods).
+
 ### DIDs as DNS Records
 
 In this scheme, we encode the [[ref:DID Document]] as multiple [DNS TXT records](https://en.wikipedia.org/wiki/TXT_record).
 Comprising a DNS packet [[spec:RFC1034]] [[spec:RFC1035]], which is then stored in the [[ref:DHT]].
 
-| Name      | Type | TTL    | Rdata                                    |
-| --------- | ---- | ------ | ---------------------------------------- |
-| _did.     | TXT  |  7200  | v=0;vm=k0,k1,k2;auth=k0;asm=k1;inv=k2;del=k2;srv=s0,s1,s2 |
-| _k0._did. | TXT  |  7200  | id=0;t=0;k=`<unpadded-b64url>`           |
-| _k1._did. | TXT  |  7200  | id=1;t=1;k=`<unpadded-b64url>`           |
-| _k2._did. | TXT  |  7200  | id=2;t=1;k=`<unpadded-b64url>`           |
-| _s0._did. | TXT  |  7200  | id=domain;t=LinkedDomains;se=foo.com     |
-| _s1._did. | TXT  |  7200  | id=dwn;t=DecentralizedWebNode;se=https://dwn.tbddev.org/dwn5 |
+| Name         | Type | TTL    | Rdata                                                        |
+| ------------ | ---- | ------ | ------------------------------------------------------------ |
+| _did.`<ID>`. | TXT  |  7200  | v=0;vm=k0,k1,k2;auth=k0;asm=k1;inv=k2;del=k2;srv=s0,s1,s2    |
```

Comment:
yess. I was confused by this initially. seems obvious in hindsight, but it's good to have it be explicit